app/server/tests/api-createCommunity.test.js

```diff
@@ -0,0 +1,234 @@
+let { CommunityFunctions } = require('./functions/communityFunctions.js');
+let { AuthFunctions } = require('./functions/authFunctions.js');
+let community = new CommunityFunctions ();
+let user = new AuthFunctions();
+let token = '';
+
+const welComTitle = "Welcome";
+const welComDescript = "Welcome to theNeighbourhood";
+const welComRules = "string";
+
+const newComTitle = "hello";
+const newComDescript = "world";
+const newComRules = "1. rules";
+const newComTags = [{
+    "tag": "Informative",
+    "count": 1
```

Comment:
This wont be allowed in the future. Its a TODO to not allow setting of count for a tag on community creation.

Reply:
okay will switch that out